api-registration/server/index.js

```diff
@@ -0,0 +1,64 @@
+require('dotenv/config');
+const pg = require('pg');
+const argon2 = require('argon2'); // eslint-disable-line
+const express = require('express');
+const ClientError = require('./client-error');
+const errorMiddleware = require('./error-middleware');
+
+const db = new pg.Pool({ // eslint-disable-line
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+
+const jsonMiddleware = express.json();
+
+app.use(jsonMiddleware);
+
+app.post('/api/auth/sign-up', (req, res, next) => {
+  const { username, password } = req.body;
+  if (!username || !password) {
+    throw new ClientError(400, 'username and password are required fields');
+  }
+
+  /* your code starts here */
+  const hashed = argon2.hash(password);
+  const sql = `
+    insert into users ("username", "hashedPassword")
+    values ($1, $2)
+    returning *
+  `;
+  const params = [username, hashed];
```

Comment:
Can you slack me a screenshot of the row in your postgres table? I don't think it contains a hashed password, but I could be wrong.

Reply:
<img width="1259" alt="Screen Shot 2022-07-19 at 3 32 55 PM" src="https://user-images.githubusercontent.com/100327736/179859672-ee59a47c-a53d-4770-8c9b-886fd3cea7b7.png">